indexer/indexer_tests/interval_index_test.cpp

```diff
@@ -28,6 +28,12 @@ struct CellIdFeaturePairForTest
   uint64_t m_cell;
   uint32_t m_value;
 };
+
+auto indexValueInserter = [](auto & values)
```

Comment:
Please rewrite this into a function.
Variables in the global scope are discouraged, especially if they are not const.